OpenRA.Mods.Common/Traits/Sound/ActorLostNotification.cs

```diff
@@ -33,6 +33,7 @@ void INotifyKilled.Killed(Actor self, AttackInfo e)
 			if (IsTraitDisabled)
 				return;
 
+			// TODO: Should this extend to controlling allied units?
```

Comment:
I think this would be to much information during battle. So I would say no and remove this comment